lib/entities/process_helper.rb

```diff
@@ -0,0 +1,50 @@
+class ProcessHelper
+  include Respondent
+
+  AVAILABLE_ACTIONS = {
+    start: 'start',
+    rules: 'rules',
+    stats: 'stats',
+    leave: 'exit',
+    save_player: 'y'
+  }.freeze
+  AVAILABLE_DIFFICULTY = %w[
+    easy
+    hard
+    expert
+  ].freeze
+
+  def initialize(player: Player.new)
+    @player = player
+  end
+
+  def setup_player
+    ask_name
+    loop do
+      @player.assign_name(input.capitalize)
+      show(@player.errors_store) if @player.errors_store.any?
+      return @player unless @player.name.nil?
+    end
+  end
+
+  def setup_difficulty
+    loop do
+      select_difficulty
```

Comment:
don't use module as an outputer
use class please.

just you know that METHOD select_difficulty - is just puts.
use correct naming